app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -83,23 +89,31 @@ You can apply for a [salaried teacher training course](/funding-and-support/sala
 
 There are 3 types of financial support you can get if you’re interested in training to teach languages and physics:
 
-* an international relocation payment worth £10,000 for languages and physics
+* an international relocation payment worth £10,000 for French, German, Spanish and physics
 * bursaries worth up to £28,000 for languages and physics
```

Comment:
Does the line about bursaries also need to change languages to French, German, Spanish?

Reply:
No cos rules different for bursaries :)